packages/core-mobile/app/screens/earn/components/Balance.tsx

```diff
@@ -232,7 +234,7 @@ export const Balance = (): JSX.Element | null => {
         </Row>
       </View>
       <View>
-        {claimableInAvax?.gt(0)
+        {claimableInAvax?.gt(0.05)
```

Comment:
the balance can be 0.000x something and we can't really claim balance in that case. adjusted the threshold here to not show the claim button in such case.